src/Configuration/RectorConfigBuilder.php

```diff
@@ -1019,6 +1022,33 @@ public function withTypeCoverageLevel(int $level): self
         return $this;
     }
 
+    /**
+     * Raise your type coverage docblock from the safest type rules
+     * to more affecting ones, one level at a time
+     */
+    public function withTypeCoverageDocblockLevel(int $level): self
+    {
+        Assert::natural($level);
+
```

Comment:
like other levelling, the flag like

```php
$this->isTypeDocblockLevel = true;
```

may be needed, to ensure not mix with use of SetList